### PR TITLE
Volvo pr Ciel day 7 part 2

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 1]
```diff
@@ -420,7 +420,7 @@ Vlov was... unrelated?
 [offset:15907]{
 -- Page 462, Offset 15907. Glued. Mods: @x, @n.
 -- ……
-...%{n}
+...
```
Comment (Volvo-hackangel) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:423]: image

File: script/Ciel/Day 7/07_01_CIEL07_7.txt [hunk 2]
```diff
@@ -125,6 +125,7 @@ Without turning around, the Executor addresses the darkness behind her.
 [sha:76ee817825696731cfce098a20ed5e920b547a3b]{
 -- Page 589, Offset 16229.
 -- 「なんだテメェ。気付いてんならそう言えっての。声をかけるタイミングが掴めなかったじゃねえか」
+// "was sat" is this a typo or a mario thing?
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_7.txt:128]: mario thing.

File: script/Ciel/Day 7/07_01_CIEL07_7.txt [hunk 3]
```diff
@@ -468,7 +469,7 @@ Mere seconds ago, a web of over seven hundred spirit strings had surrounded the
 [sha:9c6396c742c0f93d8a4e286822589d66cb360cec]{
 -- Page 614, Offset 16283.
 -- 「……魔術詠唱はなかったな。指先から魔力を流しただけで燃やしやがったか。嘘みてぇな生成量だ。並の魔術師の何百倍だ？　とっくに化け物だな代行者シエル？
-"I didn't hear any Magecraft Incantation...
+"I didn't hear any %{i}Magecraft Incantation%{/i}...
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_7.txt:472]: arc route already has roa mentioning automatic incantation so I don't think this needs italics

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 7/07_01_CIEL07_2_2.txt [hunk 4]
```diff
@@ -946,8 +946,8 @@ Senpai said she'd erase my memory after Vlov was defeated..."
 [sha:b5d381434153e9dab503e123832fd743d0e75fb7]{
 -- Page 343, Offset 15643.
 -- ……あー、できちゃうかぁ。でも、さすがにもう遅いかな？　一日や二日なら『思い出せない記憶』があってもごまかせるけど、もう四日でしょ？
-Hmm... I guess she could? But still, isn't it a bit late at this point?
- If you only erase one or two days you can play it off as someone having just forgotten what happened, but %{i}four%{/i}?
+Ahh... I guess she could? But still, isn't it a bit late at this point?
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_2_2.txt:949]: Might keep hmm

File: script/Ciel/Day 7/07_01_CIEL07_3.txt [hunk 5]
```diff
@@ -416,6 +416,7 @@ Well, so much for whatever errand she was supposed to be running.
 -- Page 382, Offset 15734.
 -- お姫様の気まぐれですっぽかされる立場に同情しないでもないが、それはともかく、
 // double negative preserved, but could be shortened if the need is so felt.
+// why the sudden change to Her Majesty? it's always been princess or dear princess
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_3.txt:419]: In this case it adds a bit of a mocking tone that I think fits well

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 6]
```diff
@@ -28,7 +28,7 @@ Alone in this deserted public space
 [sha:ff0aab2624c88d838148c05fc06f274b22b10ca7]{
 -- Page 433, Offset 15841.
 -- 「あ、きたきた！　こんばんは、時間ピッタリだね志貴！」
-"Oh, you're here! Good evening Shiki, you're right on time!"
+"Oh, you're here! Good evening, you're right on time, Shiki!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:31]: I know it matches the VA, but it just feels like the wrong order in EN to me

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 7]
```diff
@@ -138,15 +138,15 @@ Good grief...
...
+ I made sure to smash the security cameras, and even if I %{i}did%{/i} happen to bump into someone, I'd just knock them out for a bit."
 }
 [sha:3459f747b53b59c1de874ec182786312460edb8d]{
 -- Page 442, Offset 15860.
 -- 「カメラ壊しただぁ!?　さりげに凄いコト言ったな、それを力ずくって言うんだこのばか女っ！」
 // god, why can't it just be 'dumb bitch', it sounds so much better.
 "You destroyed the cameras?!
  I can't believe you just said that like it's no big deal.
- That's exactly the sort of thing that constitutes brute force, you moron!"
+ That's exactly the sort of thing that constitutes brute force, you stupid woman!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:149]: it's fine to vary ばか女 imo, it's a dumb phrase and english is replete with ways to call someone a doofus

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 8]
```diff
@@ -228,23 +228,23 @@ Seems like what Arcueid just said finally set some alarm bells ringing in my sub
 -- Page 448, Offset 15874.
 -- ……とはいっても逃げ切れる訳がないので、テクテクとアルクェイドのもとに戻る。
 Though... it's not like I'd actually be able to give her the slip,
- so I trudge back over to where Arcueid is standing.
+ so I trudge back over to where she is standing.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:231]: she's

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 9]
```diff
@@ -228,23 +228,23 @@ Seems like what Arcueid just said finally set some alarm bells ringing in my sub
...
 }
 [sha:b4f838845f5865d09dded09f17aa2c73798ce960]{
 -- Page 449, Offset 15875.
 -- 「わるい、考え事をしていたら足が勝手に動いた。
-"My bad, as I was thinking my legs took on a life of their own.
+"My bad, as I was thinking, my legs took on a life of their own.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:236]: Not a fan of this comma

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:423]: Yeah I did this because we generally don't like starting the line with a lead-in ellipsis. Honestly you could probably just make this a `%{nothing}`

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 10]
```diff
@@ -617,7 +617,7 @@ I somewhat awkwardly break eye contact with Arcueid.
 -- アルクェイドの目的が『吸血鬼殺し』だというから、完全に早とちりしていたようだ。
 // identical to first part of Offset 17176
 Because she said that killing vampires was her objective,
- I immediately jumped to what seems to have been the wrong conclusion.
+ I immediately jumped to what seemed to have been the wrong conclusion.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:620]: Seems is correct - it's clear _now_ that it was the wrong conclusion, but at the time it _seemed_ correct

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 11]
```diff
@@ -892,7 +892,7 @@ Which means... now it's time for me to hold up my end of the bargain.
 -- Page 495, Offset 15989.
 -- いいよ、俺も吸血鬼の排除には賛成だし。怖いのはイヤだけど、我慢して協力してやる」
 Why not. I fully endorse getting rid of the vampires anyway.
- It sounds terrifying, and I don't like that, but I'll bear with it and cooperate."
+ It sounds terrifying, and I don't like it, but I'll bear with it and cooperate."
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:895]: He's actually saying he doesn't like being afraid here, pretty sure - 怖いのはイヤだけど hence "I don't like that [it sounds terrifying" instead of "I don't like it [getting rid of the vampires]". Should probably just make it more clear - 
> It sounds terrifying, so I'm not looking forward to it, but...

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 12]
```diff
@@ -923,7 +923,7 @@ She's counting on me...?
 // temptress that seduces men seems a bit redundant, maybe truncate to just 'seductress'?
 // まさにその通りだ。 could be translated in a million ways, so be creative.
 I feel like there was another meaning to the word vampire...
- Maybe that a 'vamp' can also refer to a temptress that seduces men.
+ Maybe a 'vamp' can also refer to a temptress that seduces men.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:926]: Gotta keep the `that` here imo - otherwise it sounds like he's hypothesizing rather than stating what is in fact a real definition.

File: script/Ciel/Day 7/07_01_CIEL07_6.txt [hunk 13]
```diff
@@ -1031,7 +1032,7 @@ It doesn't seem like there are any other vampires here.
 [sha:c2bb4ee91c4371e4b9ee690c097ab7f7c5082653]{
 -- Page 569, Offset 16175.
 -- ロアとやらが健在ならマンションの住人すべてが襲いかかってきたかもしれないが、今はこれだけだ。
-It's possible that if this Roa person had still been alive and well the entire apartment complex would have swarmed us,
+It's possible that if this Roa person had still been alive and well, the entire apartment complex would have swarmed us,
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_6.txt:1035]: This one breaks the flow weirdly imo

File: script/Ciel/Day 7/07_01_CIEL07_6.txt [hunk 14]
```diff
@@ -1056,12 +1057,12 @@ I call out to Arcueid, and leave the garage.
 -- Page 571, Offset 16179.
 -- 何年も密閉されていた死者たちの暗闇は、これでようやく、外からの明かりに照らされる事になった。
 With this, the seal has been broken.
- Light from outside can finally illuminate the darkness of the Dead for the first time in who knows how many years.
+ Light from the outside can finally illuminate the darkness of the Dead for the first time in who knows how many years.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_6.txt:1060]: Revert

File: script/Ciel/Day 7/07_01_CIEL07_6.txt [hunk 15]
```diff
@@ -1178,7 +1179,7 @@ But I really wish she'd do something about her complete lack of common sense.
 [sha:8de85a2769b77b37e7feb8e53d999ffd3c405d6c]{
 -- Page 580, Offset 16201.
 -- ちょっとは考えろ、ばか女！」
-Stop and think for once, you dumbass!"
+Stop and think for once, you stupid woman!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_6.txt:1182]: variety is the spice of life

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 16]
```diff
@@ -203,7 +204,7 @@ Her telling me to help her out is basically the same as ordering me to use my My
 [sha:1e403030447cc06d2bf339d2f5ecfabcfe7adf04]{
 -- Page 635, Offset 16339.
 -- それを承知で、でも無理をするな、と言うアルクェイドの言い分がおかしかった。
-Arcueid is well aware of this, yet still tells me not to push myself too hard――a strange stance to take.
+She is well aware of this, yet still tells me not to push myself too hard――a strange stance to take.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:207]: She's

File: script/Ciel/Day 7/07_01_CIEL07_9.txt [hunk 17]
```diff
@@ -113,7 +113,7 @@ I surrender myself to the enveloping hand of weariness,
 -- 抗えない闇に落ちていく中、何が愚かなのかを追究しようとしたが、意識は千切れるように途絶えていった。
 As I fall into that overpowering darkness,
  I attempt to reflect on just what part of my thoughts were foolish,
- but am cut short as my consciousness is dashed to pieces.
+ but I am cut short as my consciousness is dashed to pieces.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_9.txt:116]: revert

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 18]
```diff
@@ -17,6 +17,7 @@ I finally let my legs stumble to a halt,
 [sha:04fd958e9d9cb1d19b7955c684026f85915db121]{
 -- Page 623, Offset 16306.
 -- 一方、あのお姫様はと言うと、
+// that makes it three
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:20]: I guess we can revert it if we want, but I'm not a stickler for consistency

File: script/Ciel/Day 7/07_01_CIEL07_3.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 7/07_01_CIEL07_3.txt:419]: I think dear princess also has that mocking tone but sure a little variety wont hurt

File: script/Ciel/Day 7/07_01_CIEL07_6.txt (hunk 13 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 7/07_01_CIEL07_6.txt:1035]: I thought it was too wordy and needed a break somewhere

File: script/Ciel/Day 7/07_01_CIEL07_8.txt (hunk 18 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 7/07_01_CIEL07_8.txt:20]: My vote is to revert but I'm fine keeping the first one

Comment (Volvo-hackangel) [reply script/Ciel/Day 7/07_01_CIEL07_8.txt:20]: On second thought, it's fine to leave as is at least for this entire segment

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 7 above)
Comment (Petrikow) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt]: I do think there is some level to which it's a 'trademark phrase' of Shiki's, just like him calling her アイツ (which is unfortunately much harder to capture). I think it should be consistent, but it probably shouldn't be "stupid woman" which just sounds kinda boring.

**Review by rschlaikjer — APPROVED** [review]

File: script/Ciel/Day 7/07_01_CIEL07_6.txt [hunk 19]
```diff
@@ -879,7 +879,7 @@ Still, couldn't you at least have given me a heads-up?
 [sha:338a1f27676f5f0aa7b662d96e3e6743acb9dcb4]{
 -- Page 559, Offset 16149.
 -- 「ええ、楽しんだのは事実よ。志貴の魔眼がどれだけ凄いのか、自分の目できちんと確かめたかったから」
-"Yeah, its true I did enjoy it.
+"Yeah, it's true I did enjoy it.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_6.txt:882]: it's true, I did enjoy it

**Review by Avuxo — APPROVED** [review]
